adr/0015-home-assistant-one.md

```diff
@@ -0,0 +1,82 @@
+# 0015. Installation method: Home Assistant One
+
+Date: 2020-06-08
+
+## Status
+
+Accepted
+
+## Context
+
+Define a supported installation method as per [ADR-0012](https://github.com/home-assistant/architecture/blob/master/adr/0012-define-supported-installation-method.md).
+
+## Decision
+
+Home Assistant One is the full installation of our all-inclusive home automation system. Best in class home automation is complemented with a UI for configuring your system, making backups and safe updates with automatic rollback.
+
+This is the general recommended installation method and the one that our website and documentation should be focused on.
+
+### Supported boards/hardware/machines
+
+- Raspberry Pi 2
```

Comment:
Currently the docs say _not recommended_ - if we're going to say that there, we should say it here.